File: vue/src/api/system/SystemLog.js
import request from '@/utils/request'

export function list(data) {
    return request({
        url: '/system/log/list',
        method: 'post',
        data: data
    })
}

export function query(id) {
    return request({
        url: '/system/log/query?id=' + id,
        method: 'get'
    })
}

File: springboot/src/main/java/com/hzvtc/myproject/entity/SystemLog.java
package com.hzvtc.myproject.entity;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;
import lombok.experimental.Accessors;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.Date;

/**
 * @author 熊新欣
 * @date 2020-12-29
 */
@Data
@Accessors(chain = true)
public class SystemLog {
    private Long id;

    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private LocalDateTime operateTime;

    private Long userId;

    private String clas;

    private String method;

    private String ip;

    private String param;

    private String url;

    private String result;

    private String description;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private LocalDateTime operateTimeStart;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private LocalDateTime operateTimeEnd;

    private SystemUser systemUser;
}
